main.go

```diff
@@ -154,17 +154,24 @@ func dateParamToEpoch(s string, d int64) int32 {
 	switch s {
 	case "now":
 		return int32(timeNow().Unix())
-	case "yesterday", "today", "tomorrow":
+	case "yesterday", "today", "tomorrow", "midnight", "noon", "teatime":
 		t := timeNow()
 		switch s {
 		case "yesterday":
-			t.AddDate(0, 0, -1)
-		case "tomorrow":
-			t.AddDate(0, 0, 1)
+			t = t.AddDate(0, 0, -1)
+		case "tomorrow", "midnight":
+			t = t.AddDate(0, 0, 1)
 		}
 
 		yy, mm, dd := t.Date()
 		midnight := time.Date(yy, mm, dd, 0, 0, 0, 0, defaultTimeZone)
+		switch s {
+		case "noon":
+			midnight = midnight.Add(time.Hour * 12)
```

Comment:
Will it though? I guess what I am trying to say is that 00:00 is the same for all days, DST or not, and noon/teatime are all x hours relative to 00:00.